### ISSUE TITLE
colorize items by relative level, not absolute

### ISSUE THREAD

**Issue by vgaming** [flareteam/flare-game#799]
This is a feature request to make item colors more helpful for artifact navigation.

Current situation: you don't know how useful an item is until you hover it. Colors *sometimes* help, or at least they should've been helpful, but it's mostly negated by the fact that lower-level artifacts may have the same color as very high-level.

The proposal is to make colors dependent on level difference instead. For example: weak lower-level artifacts can be made white. Higher level is green. More higher level yellow. Super high-level items and unique artifacts violet. The point is not the assignment of colors specifically, but the general idea, of course.

**Comment by dorkster** [comment]
Interesting idea. I think turning down the brightness/saturation of the colors for "weak" items would be appropriate.

Basically, we could have a threshold for how many levels below the current level is considered weak. For example, if the player's level is 8 and the modder sets the threshold to 2 levels, items with a level less than 6 would be considered weak for the player.

However, I don't think such a threshold would fit will with the Empyrean campaign's items specifically. The items are split into tiers of 4 levels each: 1-4, 5-8, 9-12, 13-16. So perhaps the engine should be able to look at these tiers and consider items in lower tiers than the player's tier to be weak.

**Comment by vgaming** [comment]
Yes, the items are indeed split into tiers, and colors follow that convention. Still, for at least my gaming experience, I don't care about tiers as much as I care about levels and "whether I can/should put this item on me". So I'm still curious how it'll feel, even if applied to Empyrean.
